prometheus-exporter/src/main/java/be/xplore/githubmetrics/prometheusexporter/jobs/JobsLabelCountsOfLastDayExporter.java

```diff
@@ -0,0 +1,101 @@
+package be.xplore.githubmetrics.prometheusexporter.jobs;
+
+import be.xplore.githubmetrics.domain.domain.Job;
+import be.xplore.githubmetrics.domain.usecases.GetAllJobsOfLastDayUseCase;
+import be.xplore.githubmetrics.prometheusexporter.ScheduledExporter;
+import be.xplore.githubmetrics.prometheusexporter.SchedulingProperties;
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+@Service
+public class JobsLabelCountsOfLastDayExporter implements ScheduledExporter {
+    private static final Logger LOGGER = LoggerFactory.getLogger(JobsLabelCountsOfLastDayExporter.class);
+    private final GetAllJobsOfLastDayUseCase getAllJobsOfLastDayUseCase;
+    private final MeterRegistry registry;
+    private final String cronExpression;
+
+    public JobsLabelCountsOfLastDayExporter(
+            GetAllJobsOfLastDayUseCase getAllJobsOfLastDayUseCase,
+            SchedulingProperties schedulingProperties,
+            MeterRegistry registry
+    ) {
+        this.getAllJobsOfLastDayUseCase = getAllJobsOfLastDayUseCase;
+        this.registry = registry;
+        this.cronExpression = schedulingProperties.jobsInterval();
+    }
+
+    private void retrieveAndExportJobs() {
```

Comment:
Do we want to make these more descriptive? the class name has become more descriptive but the function names less so

Reply:
also done!, did the same for the last days workflow run counts